### ISSUE TITLE
bug: man pages and completion is not in archives

### ISSUE THREAD

**Issue by hoshsadiq** [eza-community/eza#500]
Previously, exa included the shell completion scripts and the man pages in the archive. This doesn't seem to happen any more. Can this please be added again?

**Comment by gierens** [comment]
Which archives do you mean? The assets on the release page?

**Comment by hoshsadiq** [comment]
Yes correct. Sorry, I should have mentioned :)

**Comment by cafkafk** [comment]
duplicate of #357, will do this soon, was blocked by #482, probably next release
